crates/crates_io_docs_rs/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "crates_io_docs_rs"
+version = "0.0.0"
+license = "MIT OR Apache-2.0"
+edition = "2024"
+
+[lints]
+workspace = true
+
+[features]
+mock = ["dep:mockall"]
+
+[dependencies]
+anyhow = "=1.0.98"
+async-trait = "=0.1.88"
+crates_io_env_vars = { path = "../crates_io_env_vars" }
+http = "=1.3.1"
+mockall = { version = "=0.13.1", optional = true }
+reqwest = { version = "=0.12.15", features = ["json"] }
+serde = { version = "=1.0.219", features = ["derive"] }
+thiserror = "=2.0.12"
+tracing = "=0.1.41"
+url = "=2.5.4"
+
+[dev-dependencies]
+claims = "=0.8.0"
+serde_json = "=1.0.140"
+mockito = "=1.7.0"
+test-case = "=3.3.1"
```

Comment:
https://github.com/frondeus/test-case/issues/148

I noticed it too when I ran the tests locally 😅

Reply:
I didn't notice  😅 

reason enough to drop it?

Reply:
these two tests are small enough for me to not care, but it would be painful if this happened on the bigger application tests :D